doc/refman/RefMan-tac.tex

```diff
@@ -2688,6 +2703,63 @@ \subsection{\tt inversion \ident}
 Note that \texttt{H} has been substituted by \texttt{(LeS n m0 l)} and
 \texttt{m} by \texttt{(S m0)}.
 
+\example{Using \texorpdfstring{\texttt{inversion\_sigma}}{inversion\_sigma}}
+
+Let us consider the following inductive type of length-indexed lists,
+and a lemma about inverting equality of \texttt{cons}:
+
+\begin{coq_eval}
+Reset Initial.
```

Comment:
I've just tested adding the new printing command `Set Printing Compact Contexts.` (by @Matafou) here.
I personally like the result.